### ISSUE TITLE
TypeScript type declarations for tooltip.js 1.3.0 from npm package are wrong

### ISSUE THREAD

**Issue by robertjk** [FezVrasta/popper.js#706]
## npm package version confusion

I installed latest version of [`tooltip.js`](https://www.npmjs.com/package/tooltip.js) npm package by `npm install tooltip.js@latest`. It installed version 1.3.0.

That version has wrong TypeScript types declaration. Suprisingly [type declaration file](https://github.com/FezVrasta/popper.js/blob/master/packages/tooltip/index.d.ts) in `master` branch of your repository has proper type declarations. Even more suprisingly [`package.json`](https://github.com/FezVrasta/popper.js/blob/master/packages/tooltip/package.json) for `tooltip.js` package in your repository says it's `1.2.0` version, not `1.3.0`. Nowhere in your [releases](https://github.com/FezVrasta/popper.js/releases) I can find any information about `tooltip.js` 1.3.0 release.

I don't understand why `npm install tooltip.js` installs version `1.3.0` and why this version has missing types, where version in your repo doesn't.

## Types problems

Below is `index.d.ts` from latest (`1.3.0`) `tooltip.js` npm package.

Interface `Option` is missing `closeOnClickOutside`, `innerSelector`, `arrowSelector` properties.

Class `Tooltip` is missing [`updateTitleContent()`](https://popper.js.org/tooltip-documentation.html#Tooltip+updateTitleContent) method.

All of those missing properties and methods are available in `dist/tooltip.js` file and work properly. They are just not declared and so TypeScript compiler cannot compile the code.

```
import { Boundary, Placement, PopperOptions } from 'popper.js';

export type TitleFunction = () => string;

export type Delay = Record<'show' | 'hide', number>;

export interface Options {
  container?: HTMLElement | string;
  delay?: number | Delay;
  html?: boolean;
  placement?: Placement;
  template?: string;
  title?: string | HTMLElement | TitleFunction;
  /**
   * available options are click, hover, focus, manual
   * required to form a space delimited string
   * e.g. 'hover focus'
   */
  trigger?: string;
  boundariesElement?: Boundary | HTMLElement;
  offset?: number | string;
  popperOptions?: PopperOptions;
}

declare class Tooltip {
  constructor(reference: HTMLElement, options: Options);

  _isOpen: boolean;

  show(): void;

  hide(): void;

  dispose(): void;

  toggle(): void;
}

export default Tooltip;
```

**Comment by robertjk** [comment]
I could correct it and make a pull request myslef, but as version and files in the `master` branch are  different than what I get from npm, I have no idea to what I could apply the changes.

**Comment by FezVrasta** [comment]
I did forget to publish the commit with the version bump sorry. I'm now publishing a new version with the fixed types (I hope)

**Comment by robertjk** [comment]
Okay, thanks. When approximately I can expect for it to be published?

**Comment by FezVrasta** [comment]
few minutes

**Comment by robertjk** [comment]
Great. Thanks.

**Comment by robertjk** [comment]
By the way there is one more thing that you can correct, but I'll make another issue for it.

**Comment by FezVrasta** [comment]
released 1.3.1
